geo/src/algorithm/convex_hull/mod.rs

```diff
@@ -77,7 +80,21 @@ where
     // are to be included.
     let mut ls: Vec<Coordinate<T>> = points.to_vec();
     if !include_on_hull {
+        fn coord_compare<T: GeoNum>(c1: &Coordinate<T>, c2: &Coordinate<T>) -> Ordering {
```

Comment:
Could you use `utils::lex_cmp` for this?  It compares by `x` and then by `y` but I don't think that should affect this logic.